server/src/main/java/server/BUILD

```diff
@@ -9,13 +9,13 @@ java_library(
         "BazelServices.java",
     ],
     deps = [
-        "//server/src/main/java/server/analysis:analysis",
-        "//server/src/main/java/server/buildifier:buildifier",
-        "//server/src/main/java/server/completion:completion",
-        "//server/src/main/java/server/highlighting:highlighting",
-        "//server/src/main/java/server/starlark:starlark",
-        "//server/src/main/java/server/utils:utils",
-        "//server/src/main/java/server/workspace:workspace",
+        "//server/src/main/java/server/analysis",
+        "//server/src/main/java/server/buildifier",
```

Comment:
I had no idea you could do this, sweet

Reply:
I believe that's because if you don't specify a build target it uses a default build target that is the same name as the package.

Reply:
Sweet, I like simple syntax. I'll be sure to do this as well from now on